openapi/publicapi_spec.json

```diff
@@ -869,6 +869,15 @@
           }
         }
       },
+      "AuthorisationSummary" : {
+        "type" : "object",
+        "description" : "A structure representing information about the authentication procedure of a payment",
```

Comment:
I think that 'containing' would be more accurate than 'representing'. I also prefer 'object' to 'structure' as that's the defined `type` but not sure it's my call!

My suggestion:

> Object containing information about the authentication of the payment.

OR

> Structure containing information about the authentication of the payment.